manifest.json

```diff
@@ -194,7 +195,7 @@
           "on": "http://fauvel.archivengine.com/canvas/1v"
         }],
         "otherContent": [{
-          "@id": "http://fauvel.archivengine.com/list/1v.json",
+          "@id": "http://fauvel.archivengine.com/list/english/1v.json",
```

Comment:
@akstuhl this one, I think -- right? Good catch!

Reply:
@jamiefolsom yep!

Reply:
Great, good to merge?